app.rb

```diff
@@ -0,0 +1,151 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+require 'pry'
+class App
+  def run
+    @books = []
+    @person = []
+    @rental = []
+    puts 'Welcome to School Library App!'
+    puts "\n"
+    options
+  end
+
+  def options
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all retnals for a given person id'
+    puts '7 - Exit'
+    num = gets.chomp.to_i
+    perform_switch(num)
+    options
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def perform_switch(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    when 7
+      exit_from_menu
+    else puts 'Only digits 1 to 7 are allowed'
+    end
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    num = gets.chomp.to_i
+    case num
+    when 1
+      @person.push(create_student)
+      puts 'Student created successfully'
+    when 2
+      @person.push(create_teacher)
+      puts 'Teacher created successfully'
+    end
+  end
+
+  def create_student
+    puts 'Classroom: '
+    classroom = gets.chomp
```

Comment:
Please remove the option to input `classroom` when a user selects option `3` and wants to create a new `Student`. The user should only enter the name, age, and parent permission. Kindly see the screenshot below.